src/app/static/src/app/components/genericChart/Plotly.vue

```diff
@@ -1,6 +1,10 @@
 <template>
     <div>
-        <div ref="chart" style="width:100%; height:100%;"></div>
+        <div v-if="rendering" class="text-center mt-4" style="position: absolute; top: 0; width: 100%;">
```

Comment:
Make the spinner absolute position so it shows even if user has scrolled down through the subplots. The positioned parent for this is actually the chartContainer's parent in GenericChart